quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
+
+import (
+	"math"
+	"sync"
+)
+
+// FixedLocationer computes a set of quadrature locations and weights and stores
```

Comment:
That's not defined by the interface and can be up to the implementer. This is the sort of thing I meant by "it is assumed that the properties of the integral match the assumptions of rule.". If you're choosing a type you should know what it does.